src/lib/util/options.ml

```diff
@@ -545,7 +545,7 @@ let mk_sat_opt bottom_classes disable_flat_formulas_simplification
          match s with
          | "" -> set
          | s -> Util.SS.add s set
-      ) Util.SS.empty (Str.split (Str.regexp ",") ndo)
+      ) Util.SS.empty (Re.Str.split (Re.Str.regexp ",") ndo)
```

Comment:
Rather than use Re, which is a bit much for simply splitting a string on characters, why not use `String.split_on_char` (or re-implement it, given that it's pretty simple).